.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
How was 88 picked? Does this come from a PEP or something?

FWIW Mozilla has [these guidelines](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Python_Practices) for Python.